components/CommentForm.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+
+export default function ArtPiecesInfo({ onSubmitComment }) {
```

Comment:
```suggestion
export default function CommentForm({ onSubmitComment }) {
```
ich finde CommentForm einen besser verständlichen Namen für die Funktion. ArtPiecesInfo ist ja der andere useState.

Reply:
Wenn Du das umbenennst, musst Du den import in der anderen Datei natürlich auch umbenennen.